index.ts

```diff
@@ -146,7 +146,7 @@ export default class AuditLogPlugin extends AdminForthPlugin {
   }
 
 
-  modifyResourceConfig(adminforth: IAdminForth, resourceConfig: AdminForthResource) {
+modifyResourceConfig(adminforth: IAdminForth, resourceConfig: AdminForthResource) {
```

Comment:
@tesaide this change is not needed to implement feature, plus I think it brakes whitespace count, method declaration should be on the same level